gc/base/standard/Scavenger.cpp

```diff
@@ -3216,6 +3219,33 @@ MM_Scavenger::scavengeCompletedSuccessfully(MM_EnvironmentStandard *env)
 	return !isBackOutFlagRaised();
 }
 
+/**
+ * Save the thread environment remainder references to extensions, this is required for
+ * the master thread as it is an implicit thread and in general changes from cycle to cycle.
+ * This is done at the end of the GC cycle (or end of each phase for concurrent scavenger).
+ */
+void
+MM_Scavenger::saveTenureTLHRemainders(MM_EnvironmentStandard *env)
+{
+	_extensions->_masterThreadTenureTLHRemainderTop = env->_tenureTLHRemainderTop;
+	_extensions->_masterThreadTenureTLHRemainderBase = env->_tenureTLHRemainderBase;
+}
+
```

Comment:
rename saveTenureTLHRemainders to saveMasterThreadTenureTLHRemainders. same for restore